test-data/unit/fine-grained-modules.test

```diff
@@ -1588,6 +1588,39 @@ x = ''
 ==
 ==
 
+[case testImportStarSomethingMoved1]
```

Comment:
`1` at the end of test case name implies you either forgot to add second tests or forgot to remove this `1`.

Reply:
It was the latter, though I do also kind of like always having numbers on tests, because it means you don't need to rename the original one to add a second.